### ISSUE TITLE
Question about closed_form_factorization.py

### ISSUE THREAD

**Issue by junikkoma** [rosinality/stylegan2-pytorch#160]
Hi, 
I would first like to thank you about your fascinating work. 

I have a question about method in extracting eigenvectors.
in your code, you extract your weight matrix by codes below

```ckpt = torch.load(args.ckpt)
    modulate = {
        k: v
        for k, v in ckpt["g_ema"].items()
        if "modulation" in k and "to_rgbs" not in k and "weight" in k
 ```

I was curious about the structure, so I wrote simple code to find out.
(RH_best.pt is pt file I got from your convert_weight.py)

``` 
matrix = torch.load('../Scoliosis/RH_best.pt')
weight = matrix['g_ema']
 for items in weight.keys():
    if "modulation" in items and "to_rgbs" not in items and "weight" in items:
        print(items)
 ```
(used keys() instead of items() because I am only interested in structure, and not the actual value of weights)
and then, I got following results.
```
conv1.conv.modulation.weight
to_rgb1.conv.modulation.weight
convs.0.conv.modulation.weight
convs.1.conv.modulation.weight
convs.2.conv.modulation.weight
convs.3.conv.modulation.weight
convs.4.conv.modulation.weight
convs.5.conv.modulation.weight
convs.6.conv.modulation.weight
convs.7.conv.modulation.weight
convs.8.conv.modulation.weight
convs.9.conv.modulation.weight
convs.10.conv.modulation.weight
convs.11.conv.modulation.weight
convs.12.conv.modulation.weight
convs.13.conv.modulation.weight
```
Since there is 
```
"to_rgbs" not in k
```
 in if clause, I am guessing to_rgb1.conv.modulation.weight should also not be included as well.

Also, I am wondering whether conv1.conv.modulation.weight should be included as well.

To summarize my question, I want to know if first two weights were included as they are part of the whole layer system,
or they were included by mistakes/glitchs.

Thank you for taking care into this issue.

**Comment by junikkoma** [comment]
I have found out I totally misunderstood Closed Factorization.
Sorry for stupid question

**Comment by rosinality** [comment]
Hmm I think it is valid questions. It seems like that I didn't thought much about that, and maybe it is better to exclude to_rgb1.

**Comment by junikkoma** [comment]
Oh thanks for replying.
What's your opinion on conv1??

**Comment by rosinality** [comment]
I think it would be better to include it. But definitely it should be decided after testing. (I suspect that it will be not very different, but it is only my guess.)

**Comment by junikkoma** [comment]
Ok I will try out and share if something interesting happens :)
